tabular/src/autogluon/tabular/models/abstract/abstract_model.py

```diff
@@ -118,11 +118,22 @@ def can_infer(self) -> bool:
     def is_fit(self) -> bool:
         return self.model is not None
 
+    # TODO: v0.1 update to be aligned with _set_default_auxiliary_params(), add _get_default_params()
     def _set_default_params(self):
         pass
 
     def _set_default_auxiliary_params(self):
+        """
+        Sets the default aux parameters of the model.
+        This method should not be extended by inheriting models, instead extend _get_default_auxiliary_params.
+        """
         # TODO: Consider adding to get_info() output
+        default_auxiliary_params = self._get_default_auxiliary_params()
+        for key, value in default_auxiliary_params.items():
+            self._set_default_param_value(key, value, params=self.params_aux)
+
+    def _get_default_auxiliary_params(self) -> dict:
+        """Dictionary of auxiliary parameters that dictate various model-agnostic logic."""
         default_auxiliary_params = dict(
             max_memory_usage_ratio=1.0,  # Ratio of memory usage allowed by the model. Values > 1.0 have an increased risk of causing OOM errors.
```

Comment:
More documentation on how his parameter should be used needed.

Reply:
Added, although this is generally not meant for most users to use.